### ISSUE TITLE
slow zoom speed reflects to unzoomed randomly in 1.21

### ISSUE THREAD

**Issue by Polarite** [gliscowo/ok-boomer#9]
title explains the issue, not sure whats causing it but its happening fairly often in version 0.1.4

**Comment by gliscowo** [comment]
Do you have any method of reproducing this? I have done some experimenting myself and not been able to reproduce anything like what you're describing (if I understood that correctly, that is). 

Also, as is clearly outlined in our [contributing guidelines](https://github.com/wisp-forest/.github/blob/5dff6a3cc8567a95f8d72e3d0cc852a7f2db7364/CONTRIBUTING.md), please upload your `latest.log`. It contains a lot of information that could prove essential to successfully debugging the problem

Cheers

**Comment by theoriginalfransam** [comment]
I believe I found the theoretical problem. It's a compatibility issue with Aaron's Mod for zoom, where by default, both have their zoom key set to C, and that causes the stated issue.

**Comment by gliscowo** [comment]
Right, well, I don't think that compatibility with a different zoom mod should be of much concern (since I don't see why you'd ever need two of them). Especially if you can easily fix this by simply not binding them both to the same key (a recipe for trouble either way) I don't think anything needs to be done here

Cheers
